app/2.0/docs/upgrade.md

```diff
@@ -704,10 +704,15 @@ listener doesn't fire for the *initial* distribution.
 
 ```
 ready: function() {
+  // super.ready(); // if in a 2.0 only system
```

Comment:
Should be "for 2.0 class-based elements only." (It's not required for hybrid elements.)